components/atomic/index.js

```diff
@@ -1,12 +1,15 @@
-export { default as Figure } from "./Figure";
-export { default as Image } from "./Image";
-export { default as Video } from "./Video";
-export { default as Link } from "./Link";
-export { default as Button } from "./Button";
 export { default as SSOButton } from "./Button/patterns/SSOButton";
 export { default as Buttonish } from "./Buttonish";
-export { default as ExpandToggle } from "./ExpandToggle";
 export { default as Tile } from "./Tile";
 export { default as InvestigationTile } from "./Tile/patterns/InvestigationTile";
 export { default as SlideTile } from "./Tile/patterns/SlideTile";
 export { default as Share } from "./Share";
+
+export {
```

Comment:
replaced exports in here with epo-react-lib versions and updated all references to this file so that the individual component re-exports can be removed.